addons/video/video.c

```diff
@@ -137,6 +138,42 @@ ALLEGRO_VIDEO *al_open_video(char const *filename)
    return video;
 }
 
+ALLEGRO_VIDEO* al_open_video_f(ALLEGRO_FILE* fp)
```

Comment:
Please add a comment before this function, needed for our documentation system to work:

```c
/* Function: al_open_video_f
*/
```